dynatrace/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
Is this necessary?

Reply:
yes, this tells `mockito` to allow mocking of `final` classes and has to be enabled like this since it is an experimental feature.